app/models/katello/glue/pulp/repo.rb

```diff
@@ -199,7 +194,9 @@ def generate_importer(capsule = SmartProxy.default_capsule!)
       end
 
       def docker_feed_url(capsule = SmartProxy.default_capsule!)
-        pulp_uri = URI.parse(SETTINGS[:katello][:pulp][:url])
+        capsule_content = ::Katello::CapsuleContent.new(capsule)
+        pulp_uri = URI.parse(capsule_content.pulp_url)
```

Comment:
Given you pretty much always parse it, would a `pulp_uri` helper on `CapsuleContent` be a good idea?